tests/test_builder.rs

```diff
@@ -207,27 +205,28 @@ fn test_binary_ops() {
 
     builder.build_return(Some(&xor));
 
-    let addr = execution_engine.get_function_address("and").unwrap();
-    let and: extern "C" fn(bool, bool) -> bool = unsafe { transmute(addr) };
-    let addr = execution_engine.get_function_address("or").unwrap();
-    let or: extern "C" fn(bool, bool) -> bool = unsafe { transmute(addr) };
-    let addr = execution_engine.get_function_address("xor").unwrap();
-    let xor: extern "C" fn(bool, bool) -> bool = unsafe { transmute(addr) };
-
-    assert!(!and(false, false));
-    assert!(!and(true, false));
-    assert!(!and(false, true));
-    assert!(and(true, true));
-
-    assert!(!or(false, false));
-    assert!(or(true, false));
-    assert!(or(false, true));
-    assert!(or(true, true));
-
-    assert!(!xor(false, false));
-    assert!(xor(true, false));
-    assert!(xor(false, true));
-    assert!(!xor(true, true));
+    unsafe {
+        type BoolFunc = fn(bool, bool) -> bool;
```

Comment:
I guess it doesn't matter in this particular instance, but shouldn't the type here be a extern fn?